src/main/java/com/dougnoel/sentinel/steps/TextVerificationSteps.java

```diff
@@ -81,17 +83,17 @@ public static void verifyElementTextContains(String elementName, String assertio
             String elementText = getElement(elementName).getText();
             var expectedResult = SentinelStringUtils.format(
                     "Expected the {} element to {}{} the text {}. The element contained the text: {}",
-                    elementName, negateText, partialMatchText, text, elementText
+                    elementName, negateText, hasTooltipText, partialMatchText, text, elementText
                             .replace("\n", " "));
             log.trace(expectedResult);
             if (partialMatch) {
-                if (negate) {
+                if (negate && hasTooltip) {
```

Comment:
This only works now if you have a tooltip. This still needs to work if there is no tooltip.